### PR TITLE
Update TreeList descriptions to 18 2

### PR DESCRIPTION
**Pull request by LanaDX**

### CODE REVIEW

File: interface-elements-for-web/articles/tree-list/layout-and-navigation/reorder-columns.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Reorder Columns
+author: Bogdan Harchenko
+legacyId: 4532
+---
+# Reorder Columns
+You can drag a column header by mouse and drop it at the required position within the column header panel.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/tree-list/layout-and-navigation/reorder-columns.md]: "drag a column header by mouse" drag by mouse?

Comment (dirk-pieterse) [reply interface-elements-for-web/articles/tree-list/layout-and-navigation/reorder-columns.md]: Why "required"?

File: interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+author: Alexey Kazakov
+legacyId: 4533
+---
+# Selection and Navigation
+## Navigation
+Use the Tree List's pager to switch between pages.
+
+![ASPxTreeList_Paging](../../../images/img7367.png)
+
+To change the number of nodes displayed within a Tree List, click the **Page Size** menu and select the needed number of nodes per page.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md]: the needed number?

File: interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: Selection and Navigation
+author: Alexey Kazakov
+legacyId: 4533
+---
+# Selection and Navigation
+## Navigation
+Use the Tree List's pager to switch between pages.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md]: pager?

File: interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+![ASPxTreeList_Paging](../../../images/img7367.png)
+
+To change the number of nodes displayed within a Tree List, click the **Page Size** menu and select the needed number of nodes per page.
+
+## Selection
+
+Select and clear a check box to select and cancel the selection of the corresponding node.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md]: check box -> checkbox

Comment (dirk-pieterse) [reply interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md]: "select and cancel the selection" select and unselect?

File: interface-elements-for-web/articles/tree-list/layout-and-navigation/reorder-columns.md (hunk 1 above)
Comment (LanaDX) [reply interface-elements-for-web/articles/tree-list/layout-and-navigation/reorder-columns.md]: Perhaps: "You can drag a column header and drop it to another position in the column header panel."

File: interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md (hunk 3 above)
Comment (LanaDX) [reply interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md]: Pager - is a panel at the bottom of the treelist that enables page navigation.

File: interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md (hunk 4 above)
Comment (LanaDX) [reply interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md]: I used Microsoft style guide for choosing terminology. But the 'select and unselect' is better, I'll change. However we usually write check box.

**Review by dirk-pieterse — APPROVED** [review]

File: interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md (hunk 2 above)
Comment (dirk-pieterse) [reply interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md]: ...menu and select the number of nodes per page.

**Pull request merged**